### DISCUSSION TITLE
Different icon and colour for cancelled pipelines

### DISCUSSION THREAD

**Discussion by lonix1** [woodpecker-ci/woodpecker#2168] [Request a Feature]
### Clear and concise description of the problem

The main Woodpecker UI shows a green tick for a successful pipeline and a red check for a failed pipeline.

It also shows the red check for cancelled items - for example, when one pipeline cancels the previous one.

### Suggested solution

A cancelled pipeline should probably show a different icon with a different colour.

### Additional context

version 1.0.0
docker

### Validations

- [X] Checked that the feature isn't part of the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

**Comment by anbraten** [comment]
We previously used quite a few colors and reverted it to match the forges status checks as they only have:
- `green` (successful)
- `red` (some kind of fail, skipped, ..., so finally not executed the step) 
- and `orange` / `blue` (pending)

A cancelled pipeline is a not successfully executed pipeline and therefore kind of failed so `red`

**Reply by lonix1** [reply]
Ok, but same colour and icon for every status is confusing. For example it looks the same whether the pipeline ended in error or was cancelled manually or was cancelled automatically.

More descriptive "ideograms" for different statuses is helpful to understand something quickly.

**Reply by dr460nf1r3** [reply]
I agree that a canceled pipeline should not show as failed. Eg. if the pipeline getting canceled is actually desired (new commit got pushed or similar events), the red color unnecessarily alerts the person currently watching the last builds. A grey color would be much better to indicate this status.

**Comment by dr460nf1r3** [comment]
The answer I got while opening an issue (https://github.com/woodpecker-ci/woodpecker/issues/1946) for the exact same topic was that this is actually a [known bug](https://github.com/woodpecker-ci/woodpecker/issues/1946#issuecomment-1625314746) rather than a decision made.

**Comment by qwerty287** [comment]
should be fixed by #6011
